x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/home.spec.ts

```diff
@@ -28,7 +28,8 @@ const apisToIntercept = [
   },
 ];
 
-describe('Home page', () => {
+// flaky test
+describe.skip('Home page', () => {
```

Comment:
should we skip this? is it not fixed?

Reply:
This test is flaky. I skipped it so we can take a look during FF.